### PR TITLE
Designed the homepage

### PR DESCRIPTION
**Pull request by GregPikitis**
I think you'll like it. 
**You're welcome**:grey_exclamation: :smile: :shit:

### CODE REVIEW

**Comment by hbh7** [comment]
I am very pleased with this code. I definitely vote you use it.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+		<marquee behavior="alternate" scrollamount="500">
+				Hey!!
+		</marquee>
+		</marquee>
+		<marquee>Welcome</marquee>
+		<marquee direction="down" width="100%" height="100px" >
+				beep beep
+		</marquee>
+		<marquee direction="left" width="100%" height="100px" scrollamount="70">
+				boop boop boop boop boopboop boop boop boop boopboop boop boop boop boopboop boop boop boop boop boop boop boop boop boop boop boop boop boop boop boop boop boop
```
Comment (hbh7) [inline index.html]: beep boop boop bop

File: index.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
```
Comment (hbh7) [inline index.html]: Important

**Comment by jad340** [comment]
Although this commit looks promising, it's not exactly the direction I would personally like to go with this site. Closing.

**Comment by hbh7** [comment]
A pity. I'm not quite sure your words reflect your actions though, and I am forced to conclude you do actually like this direction. 

Current site

**Comment by jad340** [comment]
Don't tell my mom

**Comment by hbh7** [comment]
Oh of course not......

**Comment by GregPikitis** [comment]
@hbh7 Actually, I have been informed that he outsourced and hired himself to design that piece of web mastery, and it does not reference, use, or relate in any way to my original commit.

**Comment by jad340** [comment]
Don't tell my dad

**Pull request merged**